cl/people_db/templates/view_person.html

```diff
@@ -24,6 +24,68 @@
         });
     });
 </script>
+<script type="application/ld+json">
+      {
+        "@context": "http://schema.org",
+        "@type": "Person",
+        "honorificPrefix": "Judge",
+        "familyName": "{{ person.name_first }}",
+        "givenName": "{{ person.name_last }}",
+        "gender": "{% if person.gender == 'm' %}http://schema.org/Male{% else %}http://schema.org/Female{% endif %}",
+        "nationality": {
+          "@type": "Country",
+          "name": "United States of America"
+        },
+        {% if person.has_photo %}
+        "image":
+          {% with person.name_last|add:"-"|add:person.name_first|lower|slugify as img_path %}
```

Comment:
If we're going to repeat ourselves here, we should have a comment (`{# comment #}`). But what we might be able to do instead is just make the `with` statement bigger so it encompasses the entire zone where it's needed. Wouldn't be crazy just to put it at the top?

Reply:
What should the comment say?

I thought about a `with`. These two sections are in different `block`s. I thought it'd be too big a change to merge these two blocks and surround it all with a big `with`. I chose inefficiency over that. :)

Reply:
The comment would just be a heads up that this code is repeated down below, and I suppose we'd want a comment down below too. Just to keep them in sync. 

Agree re not merging the blocks. I take it you tried wrapping the blocks with a `with` statement and that didn't work?

Reply:
I guess we could also compute this variable in the view and send it to the template that way. That's the other kinda obvious solution. There's a whole hell of a lot in this `with` statement.

Reply:
@mlissner Please see latest commit to move the template logic for img.src to the view. I wasn't able to test this because I don't know how to add a photo for a person via the admin UI. 

![image](https://user-images.githubusercontent.com/480621/45849429-dd488580-bcff-11e8-8df2-f446d59e70a7.png)

Do you have some fake data I can load into my DB for cases like this?

Reply:
It's...pretty hard. The idea is that the images should be loosely bound to CourtListener, so they're in their own repo and we do all this funky stuff to essentially look them up. This looks good to me though, so I'm going to deploy it and we can test live.